cr-core/build.gradle

```diff
@@ -26,10 +26,15 @@ targetCompatibility = 1.8
 
 // We use both Maven Central and our own Artifactory instance, which contains module builds, extra libs, and so on
 repositories {
-    mavenCentral()
     maven {
+        name "Terasology Artifactory"
         url "http://artifactory.terasology.org:8081/artifactory/virtual-repo-live"
     }
+    mavenCentral()
+    maven {
+        name "JBoss Public Maven Repository Group"
+        url "https://repository.jboss.org/nexus/content/repositories/public/"
```

Comment:
suggestion: add a comment that mentions this is needed for jpastebin only. Link to developer.jboss.org/docs/DOC-11377